worker/jobTypes/S3Publish.js

```diff
@@ -93,57 +94,37 @@ class S3PublishClass {
         );
         throw new Error(`Failed pushing to prod: ${stderr}`)
       }
-      // check for json string output from mut
-      const validateJsonOutput = stdout ? stdout.substr(0, stdout.lastIndexOf(']}') + 2) : '';
 
-      // check if json was returned from mut
       try {
-        const stdoutJSON = JSON.parse(validateJsonOutput);
+        const makefileOutput = stdout.replace(/\r/g, '').split(/\n/);
+        // the URLS are always third line returned bc of the makefile target
+        const stdoutJSON = JSON.parse(makefileOutput[2]);
         const urls = stdoutJSON.urls;
-        // pass in urls to fastly function to purge cache
-        this.fastly.purgeCache(urls).then(function (data) {
-          logger.save(`${'(prod)'.padEnd(15)}Fastly finished purging URL's`);
-          logger.sendSlackMsg('Fastly Summary: The following pages were purged from cache for your deploy');
-          // when finished purging
-          // batch urls to send as single slack message
-          let batchedUrls = [];
-          for (let i = 0; i < urls.length; i++) {
-            const purgedUrl = urls[i];
-            if (purgedUrl && purgedUrl.indexOf('.html') !== -1) {
-              batchedUrls.push(purgedUrl);
-            }
-            // if over certain length, send as a single slack message and reset the array
-            if (batchedUrls.length > 20 || i >= (urls.length - 1)) {
-              logger.sendSlackMsg(`${batchedUrls.join('\n')}`);
-              batchedUrls = [];
-            }
-          }
-        });
-      } catch (e) {
-        // if not JSON, then it's a normal string output from mut
-        // get only last part of message which includes # of files changes + s3 link
-        if (stdout.indexOf('Summary') !== -1) {
-          stdoutMod = stdout.substr(stdout.indexOf('Summary'));
-        }
-      }
 
-      return new Promise((resolve) => {
-        logger.save(`${'(prod)'.padEnd(15)}Finished pushing to production`);
```

Comment:
1. Do we want to record URLs that have been purged? Ofc I'm assuming yes
2. How do we want to log these URLs to user? Do we want to have a field in the job object, purgedUrls? I don't think it makes sense to add it to the already long `logs` array field that users have to scroll thru to find info that is relevant to their needs
3. Do we want to slack them when URLs have been purged? Also for context, we are now defining a job as finishing after it has pushed to S3 AND purged URLs associated with updated content and warmed the cache (GET'd the purged URLs)